x-pack/test/security_api_integration/tests/session_idle/cleanup.ts

```diff
@@ -32,6 +33,7 @@ export default function ({ getService }: FtrProviderContext) {
     provider: AuthenticationProvider
   ) {
     log.debug(`Verifying session cookie for ${username}.`);
+    await es.indices.refresh({ index: '.kibana_security_session*' });
```

Comment:
__question:__ If I understand correctly the refresh doesn't affect the `/internal/security/me` call that goes after it, since it retrieves session by ID. Shouldn't we instead call the refresh inside `runCleanupTaskSoon`, where we need to make sure that the search operation that cleanup does relies on the latest data?

Reply:
That makes sense. I don't think we even need it in runCleanupTaskSoon though either because we have such long wait times anyhow. We're on the second iteration of the cleanup task by the time the session expires.